services/mail/pod-mail/src/main.ts

```diff
@@ -55,19 +58,36 @@ export const main = async (): Promise<void> => {
 
 export async function handleSendMail (client: MailClient, req: Request, res: Response): Promise<void> {
   // Skip auth check, since service should be internal
-  const message: SendMailOptions = req.body
-  if (message?.text === undefined) {
+  const { from, to, subject, text, html, attachments } = req.body
+  const fromAddress = from ?? config.source
+  if (text === undefined) {
     res.status(400).send({ err: "'text' is missing" })
     return
   }
-  if (message?.subject === undefined) {
+  if (subject === undefined) {
     res.status(400).send({ err: "'subject' is missing" })
     return
   }
-  if (message?.to === undefined) {
+  if (to === undefined) {
     res.status(400).send({ err: "'to' is missing" })
     return
   }
+  if (fromAddress === undefined) {
+    res.status(400).send({ err: "'from' is missing" })
+    return
+  }
+  const message: SendMailOptions = {
+    from: fromAddress,
+    to,
```

Comment:
to in 18 builds was converted to array, is it required?

Reply:
No, nodemailer accepts both string and array:
![Screenshot 2025-03-07 at 12 14 52](https://github.com/user-attachments/assets/07801b2e-4fb7-408d-9d33-9d311b2081bd)